pkgs/tools/security/exploitdb/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "exploitdb";
-  version = "2021-05-22";
+  version = "2021-05-26";
```

Comment:
```suggestion
  version = "2021-05-28";
```

Reply:
@r-ryantm is not monitoring the upstream repo. Thus, it will take some time till the PRs for the latest release are showing-up.